pyeapi/eapilib.py

```diff
@@ -250,7 +250,7 @@ def connect(self):
 
             Redefined/copied and extended from httplib.py:1105 (Python 2.6.x).
             This is needed to pass cert_reqs=ssl.CERT_REQUIRED as parameter
-            to ssl.wrap_socket(), which forces SSL to check server certificate
+            to ssl.wrap_socket()(Now changed to ssl.SSLContext.wrap_socket() as the former has been deprecated from python 3.7), which forces SSL to check server certificate
```

Comment:
you break the formatting and coding style here. flake might complain and doc building might have too. Please adjust accordingly. Also, be accurate with spaces, as it all goes into the official documentation